woocommerce/woocommerce-admin
# Update to new Snackbar component and notice store

Fixes #2445

Replaces all existing transient notices with the `Snackbar` component.  Also replaces the existing store with the `@wordpress/notices` store.

### Screenshots
<img width="849" alt="Screen Shot 2019-07-05 at 1 51 27 PM" src="https://user-images.githubusercontent.com/10561050/60700917-3ea6b180-9f2c-11e9-9330-cc9141c4ead9.png">


### Detailed test instructions:

1. Make sure things aren't broken due to dependency changes.
2. Make sure notices still work where they previously did (for example, historical data import notices).
3. Make sure notices disappear after 10 seconds and clicking on them removes them.

### Changelog Note:

Enhancement: Use core Snackbar notices for transient notifications.
